recipes/point_cloud_utils/meta.yaml

```
@@ -0,0 +1,89 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is conviniet
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "point_cloud_utils" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  url: https://github.com/fwilliams/point-cloud-utils/archive/{{ version }}.tar.gz
+  sha256: 432583d9cd15672b4cee9e812529dfcf056c6fada7d0e404e432c6acd68f972e
+  # sha256 is the prefered checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # Note: --no-deps is currently required due to https://github.com/conda/conda-build/issues/3254
+  # Once resolved, it should be removed.
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+    - pip
+    - setuptools
+    - git
+    - cmake
+    - numpy
+    - scipy
+    
+  host:
+    - {{ compiler('cxx') }}
+    - python
+    - pip
+    - setuptools
+    - git
+    - cmake
+    - numpy
+    - scipy
+    
+  run:
+    - python
+    - numpy
```

See [here](https://cf-docs-restructure.readthedocs.io/en/docs_restructure/maintainer/knowledge_base.html#linking-numpy).
```suggestion
    - {{ pin_compatible('numpy') }}
```